docs/features/walletconnect.md

```diff
@@ -0,0 +1,125 @@
+
+## Why should I care?
+
+From an end user standpoint, they really don’t need to understand _how_ it works, but they will need to accept the _connection request_ initiated by the dApp within their mobile wallet. As a dApp developer you’ll want to understand some basics about the protocol for a successful integration and provide a seamless user experience. Algorand provide a [custom WalletConnect schema](../reference/walletconnect-schema.md) and a [quick start guide](#quick-start-for-dapp-devs) for accelerating integrations.
+
+<center>![WalletConnect Header](../imgs/walletconnect-header.png)</center>
+
+The official Algorand Wallet is among the leading crypto wallets supporting the WalletConnect protocol as the industry standard for secure payment prompts and application call transactions. Importantly, user's private keys never leave their mobile device and are never exposed to connected dApps. Adding a Ledger Nano device to the Algorand Wallet further extends the user's security profile while maintaining connection simplicity enabled by the WalletConnect integration.
+
+## How it works
+
+Communication between the dApp and the mobile wallet (_peers_) happens over a _bridge server_ that relays messages without access to their content. Message contents are symmetrically encrypted through a shared key between the two peers using session data. The connection is initiated by the dApp displaying a QR Code or deep link with a standard WalletConnect URI and is established when the user approves this connection request within their Algorand Wallet. Public bridge servers are maintained by WalletConnect and dApp developers may also deploy their own.
+
+Information exchange between the peers typically consists of an unsigned atomic transaction group from the dApp to the mobile wallet, which is then inspected by the user, signed and returned back to the dApp. The dApp will submit the signed object for confirmation by the Algorand network which will effect balances and state accordingly.
+
+<center>![WalletConnect Header](../imgs/walletconnect-establishing-connection.png)</center>
+
+## Quick start for dApp devs
+
+The TypeScript sample code below should allow a dApp developer to establish a connection and request a simple payment transaction for signature.
+
+### Install
+
+```bash tab="yarn"
+yarn add @walletconnect/client algorand-walletconnect-qrcode-modal algosdk
+```
+
+```bash tab="npm"
+npm install --save @walletconnect/client algorand-walletconnect-qrcode-modal algosdk
+```
+
+### Initiate Connection
+
+```typescript
+import WalletConnect from "@walletconnect/client";
+import QRCodeModal from "@walletconnect/qrcode-modal";
+import algosdk from "@algorand/js-algorand-sdk";
```

Comment:
```suggestion
import QRCodeModal from "algorand-walletconnect-qrcode-modal";
import algosdk from "algosdk";
import { formatJsonRpcRequest } from "@json-rpc-tools/utils";
```
These package names need to be updated, and `@json-rpc-tools/utils` must be added